uint8 RED_HERO = 1
uint8 RED_ENGINEER = 2
uint8 RED_STANDARD_3 = 3
uint8 RED_STANDARD_4 = 4
uint8 RED_STANDARD_5 = 5
uint8 RED_AERIAL = 6
uint8 RED_SENTRY = 7
uint8 RED_RADAR = 9
uint8 BLUE_HERO = 101
uint8 BLUE_ENGINEER = 102
uint8 BLUE_STANDARD_3 = 103
uint8 BLUE_STANDARD_4 = 104
uint8 BLUE_STANDARD_5 = 105
uint8 BLUE_AERIAL = 106
uint8 BLUE_SENTRY = 107
uint8 BLUE_RADAR = 109

uint8 robot_id
uint8 robot_level
uint16 current_hp
uint16 maximum_hp
uint16 shooter_barrel_cooling_value
uint16 shooter_barrel_heat_limit
uint16 chassis_power_limit
uint8 power_management_gimbal_output
uint8 power_management_chassis_output
uint8 power_management_shooter_output

time stamp